build/loglevel_critical.go

```diff
@@ -0,0 +1,6 @@
+// +build dev,critical
+
+package build
+
+// LogLevel specifies a critical log level.
+var LogLevel = "critical"
```

Comment:
Is the log hierarchy be defined somewhere?

Reply:
not explicitly, though the goal is to map to the supported log levels understood by btclog